src/gt4py/backend/gt_backends.py

```diff
@@ -459,6 +465,10 @@ def visit_StencilImplementation(
                 field_attributes = {
                     "name": field_decl.name,
                     "dtype": self._make_cpp_type(field_decl.data_type),
+                    "naxes": len(field_decl.axes),
+                    "selector": tuple(
+                        axis in field_decl.axes for axis in self.impl_node.domain.axes_names
+                    ),
```

Comment:
I pass both these to the template here even though `naxes` could be determined through meta-programming logic counting the number of `true` bits in selector. Just a convenience 🤷‍♂️